### PR TITLE
Add code for review

### PR DESCRIPTION
**Pull request by topeogunleye**
- Added all my code from Todo-List project

### CODE REVIEW

File: src/style.scss [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  width: 100%;
+  padding: 0 20px;
+}
+
+.danger {
+  color: #fff;
+  background-color: #d9534f;
+  display: grid;
+  place-items: center;
+  padding: 10px 20px;
```
Comment (edwardoluyara) [inline src/style.scss:55]: - You reused this block of code in line `61 : 63` and `69 : 71 ` instead why not use the same class for all of these selectors and then add an entirely different class to style for `background-color` alone for these selectors `.danger`, `.success`, `.info` you did a great job but this is also not following the DRY principle

File: src/style.scss [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+.success {
+  color: #fff;
+  background-color: #5cb85c;
+  display: grid;
+  place-items: center;
+  padding: 10px 20px;
+}
+
+.info {
+  color: #fff;
```
Comment (edwardoluyara) [inline src/style.scss]: - And for this lines number `51`, `59`, `67` you can also do the same as the first comment

**Review by edwardoluyara — COMMENTED** [review]
In your JavaScript code you followed the best practices eg...DRY, KISS AND YAGNI

File: src/style.scss (hunk 2 above)
Comment (topeogunleye) [reply src/style.scss]: @oluyaratosin123 , Thank you :pray: for checking my code. I have made requested changes

**Comment by edwardoluyara** [comment]
## Status Approved✔, 

# Hello @topeogunleye ,

You've done a great job by adding all required  changes 👏 
This project is 💯 clear, 
Nothing else to say than its time to merge it 😊
 

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**